can-validate-validatejs.md

```diff
@@ -1,21 +1,71 @@
 @module {Object} can-validate-validatejs
 @parent can-ecosystem
-@group can-validate-validatejs.core 1 can-validate-validatejs Core
 @package ./package.json
 
-@description A plugin for CanJS that wraps any validation library to `can.validate`.
-**Can-Validate doesn't do any validation of its own** but instead provides some abstraction to your library of choice. The chosen library is registered with can-validate using a shim.
+@description Create [can-validate/validator] functions using [validate.js](https://validatejs.org/).
```

Comment:
That will be fixed when I finish https://github.com/canjs/can-validate/issues/42